content/tutorials/get-started/get-started-with-rippleapi-for-javascript.md

```diff
@@ -208,15 +228,18 @@ If you are the administrator of the `rippled` server, you can [manually request
 
 See [Reliable Transaction Submission](reliable-transaction-submission.html) for a more thorough explanation.
 
-# RippleAPI in Web Browsers
+
+
+## Use RippleAPI in a Web Browser
```

Comment:
This is another section that could be a candidate stand-alone page, but I don't like bumping the headers down to h4-level.